MetaMask/metamask-extension
# Fix the Sentry error for `eth_getBalance`

## Explanation
Fix the Sentry error for `eth_getBalance`.


## More Information
Fixes https://github.com/MetaMask/metamask-extension/issues/10551


## Screenshots/Screencaps



### Before

https://user-images.githubusercontent.com/92310504/173789548-7a09a2a9-562e-45af-b17e-debf9ed386d3.mov




### After


https://user-images.githubusercontent.com/92310504/173789772-d8e8157d-f7e2-485a-8685-37138a715488.mov





## Manual Testing Steps

1. Import a network (i.e. Fantom Opera). You can use [Chainlist](https://chainlist.org/).
2. Connect to that network.
3. Check how many accounts you have (`n` accounts).
4. Open background console.
5. Disconnect the Internet (from the Network tab).
6. Lock MM.
7. Click `Forgot Password` and wait for the error to be shown (disconnected).
8. Import your SRP and create a new password.
9. Create `n+1` accounts.
10. There is no `eth_getBalance` error in the background console.
